volttrontesting/fixtures/rmq_test_setup.py

```diff
@@ -59,7 +59,10 @@ def create_rmq_volttron_setup(vhome=None, ssl_auth=False):
     # instance can get started and shutdown multiple times within test. But restore should happen only
     # at end of instance lifetime.
     conf_backup = os.path.join(os.path.dirname(vhome),"backup_rabbitmq_conf_"+ os.path.basename(vhome))
-    shutil.copy(os.path.join(rabbitmq_config["rmq-home"],'etc/rabbitmq/rabbitmq.conf'), conf_backup)
+    try:
+        shutil.copy(os.path.join(rabbitmq_config["rmq-home"],'etc/rabbitmq/rabbitmq.conf'), conf_backup)
+    except IOError as e:
+        _log.exception("rabbitmq.conf missing from path {}".format(conf_backup))
```

Comment:
should be "rabbitmq.conf missing from path {}".format(os.path.join(rabbitmq_config["rmq-home"],'etc/rabbitmq/rabbitmq.conf')).  This can only happen when rmq was run without ssl and with default ports. Are we still running tests without ssl ?

Reply:
I'll make the change. However this behavior is seen many times specially after running certs related tests.